### PR TITLE
Improve install guide navigation

### PR DESCRIPTION
**Pull request by frankieroberto**
Current the Install guide index page uses a form with radios to route people to one of three pages.

This replaces the form with links, as those are generally better for navigation (for example can be opened in a new tab).

The "developer-friendly" install guide is renamed as "advanced", as it might not just be for developers.

It also combines the "Download the kit" page with the advanced install guide to simplify it a bit.

## Screenshots

| Before | After|
|--------|------|
| home-before | home |
| install-before | get-started |
| developer-friendly | advanced |

### CODE REVIEW

File: app/views/install/advanced.html [hunk 1]
```diff
@@ -22,17 +22,20 @@
 
     <div class="nhsuk-grid-column-two-thirds">
 
-      <h1>Developer friendly install guide</h1>
+      <h1>Advanced install guide</h1>
 
-      <p class="nhsuk-lede-text">The prototype kit is built on the <a href="http://expressjs.com/" target="_blank" rel="noopener">Express<span class="nhsuk-u-visually-hidden">opens in new window</span></a> framework, and uses the <a href="https://github.com/nhsuk/nhsuk-frontend" target="_blank" rel="noopener">NHS.UK frontend<span class="nhsuk-u-visually-hidden">opens in new window</span></a>.</p>
+      <p>The prototype kit is built on the <a href="http://expressjs.com/" rel="noopener">Express</a> framework, and uses <a href="https://github.com/nhsuk/nhsuk-frontend" rel="noopener">NHS.UK frontend</a>.</p>
```
Comment (edwardhorsford) [inline app/views/install/advanced.html:27]:
```suggestion
      <p>The NHS prototype kit is built on the <a href="http://expressjs.com/" rel="noopener">Express</a> framework, and uses <a href="https://github.com/nhsuk/nhsuk-frontend" rel="noopener">NHS.UK frontend</a>.</p>
```

File: app/views/install/index.html [hunk 2]
```diff
@@ -13,55 +13,29 @@
...
-      </form>
+      <p>Choose from:</p>
 
-      {{ backLink({
-        "href": "/",
-        "text": "Back to Home"
-      }) }}
+      <ul>
+        <li><a href="/install/simple">Install guide</a> – for those new to prototyping in code</li>
+        <li><a href="/install/advanced">Advanced install guide</a> – for developers or those familiar with prototyping in code</li>
```
Comment (edwardhorsford) [inline app/views/install/index.html:30]:
```suggestion
        <li><a href="/install/advanced">Advanced install guide</a> – for those familiar with prototyping in code</li>
```

**Review by edwardhorsford — COMMENTED** [review]
Two minor comments to start - still need to try running it locally.

File: app/views/install/advanced.html (hunk 1 above)
Comment (frankieroberto) [reply app/views/install/advanced.html:27]: @edwardhorsford Technically it’s called the NHS.UK prototype kit, which is a bit of a mouthful. Maybe we could just use 'kit' here?

Comment (edwardhorsford) [reply app/views/install/advanced.html:27]: sounds good 👍

Comment (frankieroberto) [inline app/views/install/advanced.html:27]:
```suggestion
      <p>The kit is built on the <a href="http://expressjs.com/" rel="noopener">Express</a> framework, and uses <a href="https://github.com/nhsuk/nhsuk-frontend" rel="noopener">NHS.UK frontend</a>.</p>
```

**Review by vickytnz — DISMISSED** [review]
Fixed pages to:
* remove legacy developer mode comment
* follow nhs.uk style conventions

**Comment by vickytnz** [comment]
Not a blocking change, but I wonder if the difference between the two guides are:

Choose from the:
* [step-by-step install guide](/install/simple) – for those new to prototyping in code
* [advanced install guide](/install/advanced) – for those familiar with prototyping in code

**Comment by frankieroberto** [comment]
> Not a blocking change, but I wonder if the difference between the two guides are:
> 
> Choose from the:
> 
> * [step-by-step install guide](/install/simple) – for those new to prototyping in code
> * [advanced install guide](/install/advanced) – for those familiar with prototyping in code

I like this idea. Wasn't sure if step-by-step should be hyphenated or not though - GOVUK doesn't seem to. They also front-load the main words, so it'd be "Install guide: step by step"? See [pattern](https://design-system.service.gov.uk/patterns/step-by-step-navigation/).

Have merged for now. We can continue tweaking content as follow-up PRs.

**Pull request merged**